RctGcs/droneSimulator.py

```diff
@@ -566,10 +544,10 @@ def __do_set_frequency(self, packet: RCTComms.comms.rctSETFCommand, addr: str):
 
         # Nyquist check
         for freq in frequencies:
-            if abs(freq - self.PP_options['SDR_center_freq']) > self.PP_options['SDR_sampling_freq']:
+            if abs(freq - self.PP_options[Options.SDR_CENTER_FREQ]) > self.PP_options[Options.SDR_SAMPLING_FREQ]:
```

Comment:
related to [previous comment](https://github.com/UCSD-E4E/radio_collar_tracker_gcs/pull/76/files#r1278143389)--Ideally we stop the user from sending these to the sim at all. Raising an error after the user's already connected seems like an extreme way to handle bad user input. Not that having a check here as a last resort is a bad idea though.